doc/tutorials/pynest_tutorial/part_1_neurons_and_simple_neural_networks.rst

```diff
@@ -438,7 +438,7 @@ a constant input current, and add a second neuron.
 
 ::
 
-    import pylab
+    import plt
```

Comment:
Shouldn't this be `import matplotlib.pyplot as plt`

Reply:
I will actually remove because it is not used